Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:10.9-alpine
+
+COPY . /code
+WORKDIR /code
+RUN yarn && yarn build
```

Just to clarify to people not necessarily familiar with yarn's commandline syntax. What do you think?

```suggestion
RUN yarn install
RUN yarn run build
```